pkg/telemetry/marketing/telemeter.go

```diff
@@ -0,0 +1,19 @@
+package marketing
+
+// Telemeter defines a common interface for telemetry gatherers.
+//go:generate mockgen-wrapper
+type Telemeter interface {
+	Start()
+	Stop()
```

Comment:
Could we push this to the underlying implementation and not expose here? I guess based on the telemetry storage backend the implementation should decide when to open and one to close HTTP connection, how to pool data, etc.